packages/components/src/components/ViewYAML/ViewYAML.js

```diff
@@ -15,29 +15,54 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import jsYaml from 'js-yaml';
 import classNames from 'classnames';
+import { Prism as SyntaxHighlight } from 'react-syntax-highlighter';
 
 const ViewYAML = props => {
-  const { className, dark, resource } = props;
-
-  return (
-    <div
-      className={classNames('bx--snippet--multi', className, {
-        'tkn--view-yaml--dark': dark
-      })}
-    >
-      <code>
-        <pre>{jsYaml.dump(resource)}</pre>
-      </code>
-    </div>
-  );
+  const { className, dark, resource, enableSyntaxHighlighting = false } = props;
+
+  const renderRaw = (clz, yaml) => {
+    return (
+      <div>
+        <code className={clz}>
```

Comment:
the classes were applied to the div before, is there a reason for moving them to the code element instead?

Reply:
Oh, I'm sorry, my bad.

Reply:
Fixed